### DISCUSSION TITLE
How to enable email notification in Isso?

### DISCUSSION THREAD

**Discussion by vanangamudi** [posativ/isso#958] [Q&A] [answered]
Hi, how to enable email notifications for comments so that the commenters also receive updates when another comment is posted? I don't have experience configuring smtp and also i have no idea in Isso context! I use gmail. Can I configure isso to use gmail?

```
[smtp]
host = 
port = 
to = 
from = "ISSO" <isso user's email>
username = <email server authentication user>
password = <email server authentication password>
```

**Comment by BBaoVanC** [comment, marked as answer]
Here's the Google support page on setting up SMTP: https://support.google.com/a/answer/176600?hl=en. I think Option 2 is the one you need, basically you set host to `smtp.gmail.com`, port to `587`, username is the full email address and password is your Gmail password.
